Gum/Plugins/InternalPlugins/VariableGrid/VariableReferenceLogic.cs

```diff
@@ -124,7 +124,7 @@ private void ShowFailureMessage(List<(string, GeneralResponse)> failures)
     {
         if (failures.Count > 0)
         {
-            var message = "Invalid veriable reference(s):\n";
+            string message = "";
```

Comment:
NOTE: If you accept the other PR to fix the typo, this will probably create a merge conflict that I'll fix, just let me know.

Reply:
Actually, I'm going to cancel that other PR / Branch and just consider this the fix for both